README.adoc

```diff
@@ -14,7 +14,7 @@
 :page-permalink: /guides/{projectid}
 :page-related-guides: ['cdi-intro']
 :common-includes: https://raw.githubusercontent.com/OpenLiberty/guides-common/master
-:page-seo-title: Java bean validation
+:page-seo-title: Bean Validation tutorial
```

Comment:
"validation" with a lower case is good.

Reply:
To match the spec name, I lean toward "Bean Validation".